### PR TITLE
Local dev fixes

### PR DESCRIPTION
**Pull request by ccorda**

### CODE REVIEW

File: .githooks/post-merge [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function changed {
+  git diff --name-only HEAD@{1} HEAD | grep "^$1" > /dev/null 2>&1
+}
+
+if changed 'package-lock.json'; then
+  echo "📦 package-lock.json changed. Run npm install to bring your dependencies up to date."
```
Comment (kylehotchkiss) [inline .githooks/post-merge:6]: Why not check for yarn.lock and run yarn to update deps?

**Review by kylehotchkiss — APPROVED** [review]
Looks good but please see the husky task that's checking package-lock.json instead of yarn.lock and recommending `npm install`

File: .githooks/post-merge (hunk 1 above)
Comment (ccorda) [reply .githooks/post-merge:6]: Yeah, actually have that in a need to update version: https://github.com/patronage/spiegelworld/blob/main/website/.githooks/post-merge.sh

But it still wasn't working, flagged for a future issue #53

**Pull request merged**